src/test/groovy/org/arquillian/spacelift/gradle/SpaceliftToolBinaryTest.groovy

```diff
@@ -152,7 +152,7 @@ class SpaceliftToolBinaryTest {
         project.spacelift {
             tools {
                 ant {
-                    command ([linux:"ant", windows:"ant.bat"])
+                    command ([linux:"ant", windows:"ant.bat", mac:'ant'])
```

Comment:
I was not able to run the tests on my mac without this fix so I included it into this PR.